报错Error occurred while send

2023/01/23 21:27:48 Error occurred while send, retrying: unexpected send handshake reply panic: send retry limit exceeded.

试了一下，可以正常登陆，打开网页，但是打开第二个网页就程序就退出了，我然后试着保持页面打开不打开新页面，可以看到网页里的 xhr 请求是正常的，但是一打开新页面程序就退出了。很可能是上面说的并发连接数的关系。

终端里显示的log是这样的
```
2023/01/24 14:19:27 socks dial: 10.68.0.1:80
2023/01/24 14:19:27 socks dial: 10.68.0.1:80
2023/01/24 14:19:27 socks dial: 10.68.0.1:80
2023/01/24 14:19:27 Error occurred while recv, retrying: EOF
2023/01/24 14:19:27 socket: connected to:  x.x.x.x:8443
2023/01/24 14:19:27 tls: connected to:  x.x.x.x:8443
2023/01/24 14:19:27 recv handshake: wrote 64 bytes
00000000  06 00 00 00 36 37 61 64  37 62 31 63 32 32 31 63  |....67ad7b1c221c|
00000010  66 62 34 37 31 64 63 36  61 36 37 39 63 31 36 63  |fb471dc6a679c16c|
00000020  66 37 31 00 30 38 36 30  31 30 35 61 32 39 36 35  |f71.0860105a2965|
00000030  33 62 66 38 00 00 00 00  00 00 00 00 03 01 00 02  |3bf8............|
2023/01/24 14:19:27 Error occurred while recv, retrying: EOF
2023/01/24 14:19:27 socket: connected to:  x.x.x.x:8443
2023/01/24 14:19:27 tls: connected to:  x.x.x.x:8443
2023/01/24 14:19:27 recv handshake: wrote 64 bytes
00000000  06 00 00 00 36 37 61 64  37 62 31 63 32 32 31 63  |....67ad7b1c221c|
00000010  66 62 34 37 31 64 63 36  61 36 37 39 63 31 36 63  |fb471dc6a679c16c|
00000020  66 37 31 00 30 38 36 30  31 30 35 61 32 39 36 35  |f71.0860105a2965|
00000030  33 62 66 38 00 00 00 00  00 00 00 00 03 01 00 02  |3bf8............|
2023/01/24 14:19:27 Error occurred while recv, retrying: EOF
2023/01/24 14:19:27 socket: connected to:  x.x.x.x:8443
2023/01/24 14:19:27 tls: connected to:  x.x.x.x:8443
2023/01/24 14:19:27 recv handshake: wrote 64 bytes
00000000  06 00 00 00 36 37 61 64  37 62 31 63 32 32 31 63  |....67ad7b1c221c|
00000010  66 62 34 37 31 64 63 36  61 36 37 39 63 31 36 63  |fb471dc6a679c16c|
00000020  66 37 31 00 30 38 36 30  31 30 35 61 32 39 36 35  |f71.0860105a2965|
00000030  33 62 66 38 00 00 00 00  00 00 00 00 03 01 00 02  |3bf8............|
2023/01/24 14:19:27 Error occurred while recv, retrying: EOF
2023/01/24 14:19:27 socket: connected to:  x.x.x.x:8443
2023/01/24 14:19:27 tls: connected to:  x.x.x.x:8443
2023/01/24 14:19:27 recv handshake: wrote 64 bytes
00000000  06 00 00 00 36 37 61 64  37 62 31 63 32 32 31 63  |....67ad7b1c221c|
00000010  66 62 34 37 31 64 63 36  61 36 37 39 63 31 36 63  |fb471dc6a679c16c|
00000020  66 37 31 00 30 38 36 30  31 30 35 61 32 39 36 35  |f71.0860105a2965|
00000030  33 62 66 38 00 00 00 00  00 00 00 00 03 01 00 02  |3bf8............|
2023/01/24 14:19:27 Error occurred while recv, retrying: EOF
panic: recv retry limit exceeded.

goroutine 38 [running]:
EasierConnect/core.StartProtocol.func1()
        EasierConnect/core/protocol.go:196 +0x105
created by EasierConnect/core.StartProtocol
        EasierConnect/core/protocol.go:199 +0xd2
```

--- a/main.go
+++ b/main.go
@@ -5,6 +5,7 @@ import (
 	"flag"
 	"fmt"
 	"log"
+	"runtime"
 )
 
 func main() {
@@ -51,4 +52,6 @@ func main() {
 	log.Printf("Login success, your IP: %d.%d.%d.%d", ip[0], ip[1], ip[2], ip[3])
 
 	client.ServeSocks5(socksBind, debugDump)
+
+	runtime.KeepAlive(client)
 }

